src/java/picard/illumina/IlluminaBasecallsToFastq.java

```diff
@@ -88,9 +118,9 @@
     @Option(doc = "Lane number. ", shortName = StandardOptionDefinitions.LANE_SHORT_NAME)
     public Integer LANE;
 
-    @Option(doc = "The prefix for output fastqs.  Extensions as described above are appended.  Use this option for " +
-            "a non-barcoded run, or for a barcoded run in which it is not desired to demultiplex reads into separate " +
-            "files by barcode.",
+    @Option(doc = "The prefix for output FASTQs.  Note, you should actually use a file path here to direct the output to a location outside of the working directory.  " +
```

Comment:
Sorry the meaning of prefix escaped me when I wrote this.  I will just eliminate this sentence.